sormas-app/app/src/main/java/de/symeda/sormas/app/backend/person/Person.java

```diff
@@ -173,6 +174,8 @@
 	private String passportNumber;
 	@Column
 	private String nationalHealthId;
+	@Enumerated(EnumType.STRING)
+	private SymptomJournalStatus symptomJournalStatus;
```

Comment:
@MartinWahnschaffeSymeda is this something we expect to receive from other 3rd party systems as well? or only from PIA ? if it is only from PIA i suggest we move this to a separate table, for example PiaRegistration and do a join with this table when needed
this way we could keep the domain entity clean from 3rd party app influences

Reply:
I guess we may, in the future, also get a status from other symptom journals, for example from climedo. I don't know if they will always have the same status types (ACCEPTED, REJECTED.. as defined in the enum), though. I would also like Martins opinion about it.

Reply:
@MateStrysewske also thinks that this should be generic.

Reply:
Yes, I agree - should be generic for now.